docs/devel/host-devices-and-device-plugins.md

```diff
@@ -0,0 +1,65 @@
+## Host Devices Assignment
+
+KubeVirt provides a mechanism for assigning host devices to a virtual machine. This mechanism is generic and allows various types of PCI devices, such as GPU or any other devices attached to a PCI bus, to be assigned. It also allows Mediated devices, such as pre-configured virtual GPUs to be assigned using the same mechanism.
+
+### Permitting Host Devices to be used in the cluster:
+
+Administrators can control which host devices will be permitted for use in the cluster
+Permitted host devices in the cluster will need to be listed in KubeVirt CR by its `vendor:product` selector for PCI devices or mediated device names
```

Comment:
Isn't it vendor/product ? I mean slash instead of colon, between vendor and product.

Reply:
Possibly worth mentioning it refers to the `resourceName` attribute.

Reply:
> Isn't it vendor/product ? I mean slash instead of colon, between vendor and product.

It expects a colon. As in pciVendorSelector: "10DE:1EB8"

Reply:
Ah, seems I didn't understand this; Then I'd say referring the parameter to which this applies is important (I thought it applied to `resourceName`.

I'd also like to point out that the `mdevNameSelector` example below does not have a colon separator: 
`- mdevNameSelector: "GRID T4-1Q"`; here you say this is colon separated.

Reply:
> Ah, seems I didn't understand this; Then I'd say referring the parameter to which this applies is important (I thought it applied to `resourceName`.
> 

np 


> I'd also like to point out that the `mdevNameSelector` example below does not have a colon separator:
> `- mdevNameSelector: "GRID T4-1Q"`; here you say this is colon separated.

Right, this is because mdevs are identified by their names: ```... `vendor:product` selector for PCI devices or mediated device names```
For example: ```cat /sys/bus/mdev/devices/[UUID]/mdev_type/name
GRID RTX8000-1Q```

Reply:
Ah! Now I get it - the selector is used for pci, the mediated device name for mediated devices. Which is exactly what you say.